asciidoc/modules/con_mg_miscellaneous-changes-in-vertx.adoc

```diff
@@ -0,0 +1,60 @@
+[id="miscellaneous-changes-in-vertx_{context}"]
+= Miscellaneous changes in {VertX}
+
+The following section describes miscellaneous changes in {VertX} {4}.
+
+== Removed the `Starter` class
+
+The `Starter` class has been removed. Use the `Launcher` class instead to start your {VertX} applications without the `main()` method.
+
+== Isolated deployment for Java 8
+
+{VertX} {4} supports Java 11. This Java version does not support isolated class loading.
+In {VertX} {4}, isolated class loading will be supported for Java 8. However, the results may not be as expected.
```

Comment:
@vietj Do we need to write the whole note from wiki or just say:
Since Java 11 does not support class loading, we cannot support it in Vert.x 4.0?

https://github.com/vert-x3/wiki/wiki/4.0.0-Deprecations-and-breaking-changes#%EF%B8%8F-isolated-deployment-for-java-8

Reply:
I think you don't need _However, the results may not be as expected._ Besides I think the current note as written is correct.

Reply:
@vietj Removed the following sentence from the note:
However, the results may not be as expected.